analytic_category/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "version": "12.0.1.0.0",
     "author":  "Eficent",
     "category": "Analytic",
-    "depends": ["analytic"],
+    "depends": ["analytic", "account"],
```

Comment:
```suggestion
    "depends": ["account"],
```

Reply:
analytic does not depend on account

Reply:
it's the contrary, `account` depends of `analytic`, so putting `account` here already implies `analytic`.

Reply:
I believe you